WalletWasabi/WabiSabi/Models/Error.cs

```diff
@@ -3,5 +3,6 @@ namespace WalletWasabi.WabiSabi.Models;
 public record Error(
 	string Type,
 	string ErrorCode,
-	string Description
+	string Description,
+	string Data
```

Comment:
I would remove the description (I know all the excuses, don't worry) and would make `Data` an `ExceptionData` object.

Reply:
I will remove it in another PR. Changed `Data` to `ExceptionData` type.